module-services/service-desktop/WorkerDesktop.cpp

```diff
@@ -113,95 +115,126 @@ void WorkerDesktop::reset()
 
 bool WorkerDesktop::handleMessage(uint32_t queueID)
 {
+    auto result       = false;
```

Comment:
I think we should use `auto` more conservatively. This is a very useful language feature to improve the readability when some long type would otherwise be necessary but for short and known types I think we should refrain from using it. And here is the edge case where `auto` is used to replace `bool` so a known type with name of the same length as `auto`.

Reply:
done